### PR TITLE
Initial test

### PR DESCRIPTION
**Pull request by Ashwin-Pokharel**
Initial selenium testing  , update .env variables and then run test.sh script

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -17,11 +17,12 @@ sleep 7
 
 # Migrate the database
 # Collect all static content
-rm -rf www/
+sudo rm -rf www/
```
Comment (kpachhai) [inline run.sh]: Is there a reason why this file was changed at all?

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+sudo rm -rf www/
+python3 manage.py collectstatic --no-input
+
+python3 manage.py makemigrations
+python3 manage.py migrate
+
+# Finally, run the Django server
+python3 manage.py test login
+
+python3 manage.py runserver 0.0.0.0:8000
```
Comment (kpachhai) [inline test.sh]: What's the point of this last command? Do we really need to run the server to test our stuff?

File: run.sh (hunk 1 above)
Comment (Ashwin-Pokharel) [reply run.sh]: i had to include that as not including it resulted in an access denied error when removing the files

File: test.sh (hunk 2 above)
Comment (Ashwin-Pokharel) [reply test.sh]: we do not , Ill change and update it

**Pull request merged**